pool.go

```diff
@@ -65,6 +66,14 @@ func (p *Pool) Public() crypto.PublicKey {
 
 // NewPool creates a pool of Keys of size n.
 func NewPool(n int, modulePath, tokenLabel, pin, privateKeyLabel string) (*Pool, error) {
+	// Some modules may block their thread during operations, so it's important to
+	// have at least as many threads as sessions in the pool to achieve full
+	// throughput.
+	if runtime.GOMAXPROCS(0) < n {
+		return fmt.Errorf("pkcs11key.NewPool: GOMAXPROCS (%d) is lower than number of "+
```

Comment:
This needs to be `return nil, fmt.Errorf("...")` - currently failing to build with an error about not returning enough arguments.